guides/common/modules/proc_calling-the-api-in-python.adoc

```diff
@@ -152,10 +152,10 @@ except ImportError:
     print("Please install the python-requests module.")
     sys.exit(-1)
 
-SAT = "{foreman-example-com}"
+{project-allcaps} = "{foreman-example-com}"
 # URL for the API to your deployed {ProjectX} server
-{project-allcaps}_API = f"https://\{SAT}/api/"
-KATELLO_API = f"https://\{SAT}/katello/api/v2/"
+{project-allcaps}_API = f"https://{{project-allcaps}}/api/"
```

Comment:
```suggestion
{project-allcaps}_API = f"https://{{project-allcaps}}/api/"
```
I know this is a bit out of scope, but can you please fix the PROJECT_API path on the line no. 38?

Reply:
I have applied Ewoud's suggestion first and cannot find "PROJECT_API" anymore. The variables are now called "FOREMAN_API" and "KATELLO_API" to reference both API endpoints. THE URL is always branded to "{project-example-com}". Is that OK @Lennonka ?